docs/markdown/authoritative/backend-generic-postgresql.md

```diff
@@ -0,0 +1,53 @@
+# Generic PostGreSQL backend
+|&nbsp;|&nbsp;|
+|:--|:--|
+|Native|Yes|
+|Master|Yes|
+|Slave|Yes|
+|Superslave|Yes|
+|Autoserial|Yes (v3.1 and up)|
+|Case|All lower|
+|DNSSEC|Yes (set `gpgsql-dnssec`)|
+|Disabled data|Yes (v3.4.0 and up)|
+|Comments|Yes (v3.4.0 and up)|
+|Module name | pgsql|
+|Launch name| pgsql|
+
+The default setup conforms to the following schema, which you should add to a PostgreSQL database.
+
+```
+!!include=../modules/gpgsqlbackend/schema.pgsql.sql
+```
+
+`zone2sql` with the `--gpgsql` flag also assumes this layout is in place.
+
+This schema contains all elements needed for master, slave and superslave operation.
+
+For full migration notes, please see [Migration](migration.md).
+
+With PostgreSQL, you may have to run `createdb powerdns` first and then connect to that database with `psql powerdns`, and feed it the schema above.
```

Comment:
default dbname (as said below) is `pdns`, best to change the psql sample command here to match.